drivers/zooz/zooz-zen32-scene-ctlr.groovy

```diff
@@ -64,17 +64,17 @@ import groovy.transform.Field
 ]
 
 // color name to parameter value mapping:
-@Field static Map<String,Integer> colorNameMap = [
-   "white": 0,
-   "blue": 1,
-   "green": 2,
-   "red": 3,
-   "magenta": 4,
-   "yellow": 5,
-   "cyan": 6
+@Field static TreeMap<Integer,String> colorNameMap = [
```

Comment:
This was changed around so it could be used as the constraints of the command and everything else in the code. TreeMap keeps is sorted.